_site/index.html

```diff
@@ -565,7 +619,7 @@ <h3>T-Z</h3>
             </p>
             <p class="word__breakdown">Gaslighting reinfoces systems of power, as gaslighting abusers will tend to be in systematically dominant positions where they are intrinsically believed over their victim (e.g., white person over person of colour, man over woman, abled person over disabled person, parent over child, manager over worker, etc). Gaslighting is an abusive and oppressive behaviour and tactic.<br><br>The abuser can become in control of the victim, who now no longer trusts their own perception. The victim can become codependent upon their abuser, and internalise their abuser's perspective, which can lead to self-doubt and self-hate.</p>
```

Comment:
This needs to be written to reflect the word in question; I think this is pasted from Gaslighting.